### PR TITLE
Redshift server and store dockerization

### PR DESCRIPTION
**Pull request by PaulMTT**
This uses the Socrata customizations as demonstrated in rapid deploy:
```
<quarkus.jib.base-jvm-image>socrata/runit-java-focal:17</quarkus.jib.base-jvm-image>
  <quarkus.jib.jvm-entrypoint>/sbin/my_init</quarkus.jib.jvm-entrypoint>
  <quarkus.jib.user>root</quarkus.jib.user>
  <quarkus.jib.working-directory>/home/socrata</quarkus.jib.working-directory>
```

Both the redshift server and store are packaged into images.
Refer to each projects `src/main/jib` for supporting files such as the run script.

### CODE REVIEW

**Comment by PaulMTT** [comment]
And demonstrating the images run as expected:

> ➜  soql-redshift-adapter git:(paulm/jib) docker run paulmaksimovich/soql-redshift-adapter-store:1.0.0-SNAPSHOT
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
*** Running /etc/my_init.d/10_syslog-ng.init...
Jan  9 16:48:38 a15e84e14d07 syslog-ng[14]: syslog-ng starting up; version='3.25.1'
*** Running /etc/my_init.d/clortho-get...
DEBUG: AWS_REGION unset. Defaulting to us-west-2
INFO: Either CLORTHO_BUCKET or CLORTHO_PATH unset. Skipping creds download
*** Running /etc/my_init.d/set_jmx_hostname...
*** Running /etc/my_init.d/set_local_dev_hostname...
*** Booting runit daemon...
*** Runit started as PID 37
runsv store-redshift: warning: ./log: not a directory.

exec /sbin/setuser socrata java -agentlib:jdwp='transport=dt_socket,server=y,suspend=n,address=*:5005' -Dquarkus.profile=docker -Djava.util.logging.manager=org.jboss.logmanager.LogManager -jar /home/socrata/quarkus-run.jar
Picked up JAVA_TOOL_OPTIONS: -Dcom.sun.management.jmxremote.port=11114 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false
Listening for transport dt_socket at address: 5005
__  ____  __  _____   ___  __ ____  ______
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/
2024-01-09 16:48:43,840 WARN  [io.agr.pool] (agroal-11) Datasource '<default>': Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
2024-01-09 16:48:43,853 WARN  [org.hib.eng.jdb.env.int.JdbcEnvironmentInitiator] (JPA Startup Thread) HHH000342: Could not obtain connection to query metadata: org.postgresql.util.PSQLException: Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.

File: server-redshift/src/main/jib/etc/service/server-redshift/run [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Turn verbose output OFF so we don't echo the output of the secrets file
+set +ev
+
```
Comment (zachlefevre) [inline server-redshift/src/main/jib/etc/service/server-redshift/run:5]: we will need secrets and I suggest adding
```
if [ -f /dev/shm/redshift-secondary-env ]; then
  source /dev/shm/redshift-secondary-env
else
  echo "/dev/shm/redshift-secondary-env not found. Check your Clortho config."
fi
```

File: store-redshift/src/main/jib/etc/service/store-redshift/run [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Turn verbose output OFF so we don't echo the output of the secrets file
+set +ev
+
```
Comment (zachlefevre) [inline store-redshift/src/main/jib/etc/service/store-redshift/run:5]: same comment as above

File: server-redshift/src/main/jib/etc/service/server-redshift/run (hunk 1 above)
Comment (rlvoyer) [reply server-redshift/src/main/jib/etc/service/server-redshift/run:5]: How is Rapid Deploy doing secrets management? I thought they were using AWS secrets manager. Any reason we are not doing the same?

Comment (zachlefevre) [reply server-redshift/src/main/jib/etc/service/server-redshift/run:5]: https://github.com/socrata/rapid-deploy/blob/main/service/src/main/jib/etc/service/rapid-deploy/run#L7

They are using Clortho.

Comment (zachlefevre) [reply server-redshift/src/main/jib/etc/service/server-redshift/run:5]: as far as I can glean from the code. They definitely source clortho in their docker runfile.

Comment (PaulMTT) [reply server-redshift/src/main/jib/etc/service/server-redshift/run:5]: Added!

**Pull request merged**